.github/workflows/ci.yaml

```diff
@@ -43,6 +43,25 @@ jobs:
         with:
           args: --build-tags containers_image_openpgp,exclude_graphdriver_btrfs,exclude_graphdriver_devicemapper,exclude_graphdriver_overlay
 
+  web-tests:
+    name: Web unit tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ./web/.nvmrc
+      - name: Install dependencies
+        run: |
+          cd web
+          npm install
+      - name: Run web unit tests
+        run: |
+          cd web
+          npm run test:unit
+
```

Comment:
We can definitely do this as follow up PR but we should probably add this to - https://github.com/replicatedhq/embedded-cluster/blob/dae1d779892acf54c117fdcedf3779cddd150335/.github/workflows/ci.yaml#L864-L880 - so that it blocks CI in case of failure.

Reply:
Also should we add the linting here too (or maybe in a separate job) `npm run lint`?

Reply:
good point. i'll add this to a follow up pr